GitPrompt.ps1

```diff
@@ -82,6 +82,9 @@ $global:GitPromptSettings = New-Object PSObject -Property @{
 
     AutoRefreshIndex                            = $true
 
+    # Valid values are "Full", "Compact", and "Minimal"

```

Comment:
You don't need to use an enum but once we drop support on master for PS v2, you can use:
```powershell
[ValidateSet('Compact','Full','Minimal')]
$BranchBehindAndAheadDisplay = "Full"
```
PS v2 support can't be dropped soon enough.  :-)